src/registrar/tests/data/test_contacts.txt

```diff
@@ -0,0 +1,7 @@
+TESTUSER|52563_CONTACT_GOV-VRSN|919-000-0000||918-000-0000||testuser@gmail.com|GSA|VERISIGN|ctldbatch|2021-06-30T17:58:09Z|VERISIGN|ctldbatch|2021-06-30T18:18:09Z|
```

Comment:
**suggestion (blocking)**: initials and last names are most definitely PII. These column 1 usernames should absolutely not be any that occur in the real data.

Reply:
This was pulled straight from test data.  Should not be real.  If so, we need to update the test files provided in our cloud folder.  For now, I can further obfuscate it.

Reply:
(We spoke about this on slack.  Resolving in next push)